src/test/groovy/unit/org/pih/warehouse/invoice/PrepaymentInvoiceServiceSpec.groovy

```diff
@@ -50,52 +52,31 @@ class PrepaymentInvoiceServiceSpec extends Specification implements ServiceUnitT
         null != service.generateInvoice(order)
     }
 
-    void 'PrepaymentInvoiceService.generateInvoice() should create invoice not including already invoiced items'() {
+    void 'PrepaymentInvoiceService.createFromShipmentItem() should calculate quantity to invoice: #quantityToInvoice when quantity is: #quantity and quantity invoiced: #quantityInvoiced'() {
```

Comment:
not worth testing generatePrepaymentInvoice() (or whatever) here?

Reply:
To be honest there were so many breaking changes, that I was just only fixing what was not working. I am going to add a ticket to revisit tests after the whole implementation on the backend is sealed after QA, because going back and forth on these tests at this stage is frustrating